### ISSUE TITLE
📢Reminder: Submit PRs Only After Raising and Being Assigned to Issues

### ISSUE THREAD

**Issue by Tejashri-Taral** [Tejashri-Taral/TrendTrove-Ecommerce#90]
### Discussed in https://github.com/Tejashri-Taral/TrendTrove-Ecommerce/discussions/89

<div type='discussions-op-text'>

<sup>Originally posted by **Tejashri-Taral** October  3, 2024</sup>
Hi everyone,

I’ve noticed some contributors are **submitting pull requests (PRs) without first raising an issue or getting assigned to one.**
To keep things organized and in line with the **project guidelines.**

Read How To Contribute and Guidelines:
[Click Here to View](https://github.com/orgs/GSSoC24/discussions/175)

### I want to remind everyone of the following steps:
- **Raise an Issue**: Before working on any changes, make sure you create an issue describing the problem or feature you want to address.

- **Get Assigned:** Wait for the project admins (PAs) to assign the issue to you. Only after you’ve been assigned should you start working on a pull request.

- **Link the Issue in Your PR:** Once assigned, include the issue number in your pull request description for easy reference.

Submitting a PR without following these steps may lead to your work being put on hold, especially if someone else is already assigned to the issue. 
**We want to avoid duplication and ensure fair contribution opportunities for everyone.**

If you’ve submitted a PR without raising or being assigned to an issue, please create an issue now, and we’ll proceed accordingly.

**Thank you for your understanding and cooperation!**</div>
